src/main/java/org/apache/commons/math4/util/OpenIntToFieldHashMap.java

```diff
@@ -566,6 +590,9 @@ public T value()
          * @exception ConcurrentModificationException if the map is modified during iteration
          * @exception NoSuchElementException if there is no element left in the map
          */
+        @SuppressWarnings({"index:assignment.type.incompatible", "index:array.access.unsafe.low", "index:array.access.unsafe.high", "index:unary.increment.type.incompatible"}) /*
+        #1: These statement is never executed with next = states.length because when next = states.length - 1, the try catch block makes next = -2
```

Comment:
It's not clear what statement this refers to - I presume it is the `++next` expression.